static/Projects.html

```diff
@@ -0,0 +1,51 @@
+<html>
+<head>
+<title>
+  Just a Ruby Girl, In a Ruby World...</title>
+</head>
+<header>
+  <a href="/home">Home</a>
+  <a href="/about">About</a>
+  <a href="/blog">Blog</a>
+  <a href="/projects">Projects</a>
+</header>
+
+<body>
```

Comment:
I totally forgot to cover the importance of the `<body>` tag in class. :( I'm sorry about that.

The short rule is that all of your html code needs to be either nested in `<head>` or in `<body>`. Moreso, anything thing that's supposed to be visible on the page must be nested inside `<body>`. So your basic structure should look something like...

```html
<html>
  <head></head>
  <body>
    <header></header>
    All your other content in here somewhere.
    <footer></footer>
  </body>
</html>
```